src/System.CommandLine/Argument{T}.cs

```diff
@@ -45,6 +66,13 @@ public Argument(Func<T> getDefaultValue) : this()
             SetDefaultValueFactory(() => getDefaultValue());
         }
 
+        /// <summary>
+        /// Initializes a new instance of the Argument class.
+        /// </summary>
+        /// <param name="name">The name of the argument.</param>
+        /// <param name="parse">A custom argument parser.</param>
+        /// <param name="isDefault"><c>true</c> to use parser result as default value.</param>
```

Comment:
This should probably include a reference to the `parse` parameter on this constructor, as it might be a little ambiguous otherwise.

Reply:
Changed it, hopefully this is what you meant.